docs/sources/setup-grafana/configure-docker.md

```diff
@@ -8,45 +8,70 @@ keywords:
+The Alpine variant is highly recommended when security is important and you want an  image size that’s as small as possible. Note that the Alpine variant uses [musl libc](http://www.musl-libc.org/) instead of [glibc and friends](http://www.etalabs.net/compare_libcs.html), so certain software might run into issues depending on the depth of their libc requirements. However, most software doesn’t have an issue with this, so this variant is usually a very safe choice.
+
+## Ubuntu image
+
+**Grafana Enterprise**: `grafana/grafana-enterprise:<version>-ubuntu`
 
-### Run Grafana container using bind mounts
+**Grafana Open Source**: `grafana/grafana-oss:<version>-ubuntu`
 
-You may want to run Grafana in Docker but use folders on your host for the database or configuration. When doing so, it becomes important to start the container with a user that is able to access and write to the folder you map into the container.
+These images are based on [Ubuntu](https://ubuntu.com/), available in the [Ubuntu official image](https://hub.docker.com/_/ubuntu). It is an alternative image for those who prefer an Ubuntu-based image and/or are dependent on certain tooling not available for Alpine.
+
+## Run a specific version of Grafana
+
+You can also run a specific version, or run a beta version based on the main branch of the [grafana/grafana GitHub repository](https://github.com/grafana/grafana).
+
+> **Note:** If you are on a Linux system (e.g. Debian, Ubuntu), you might need to add `sudo` before the command or add your user to the `docker` group by following the official documentation to [run Docker without a non-root user](https://docs.docker.com/engine/install/linux-postinstall/).
+
+
+To run a specific version of Grafana mention it in the command <version number> section:
 
 ```bash
-mkdir data # creates a folder for your data
-ID=$(id -u) # saves your user id in the ID variable
+docker run -d -p 3000:3000 --name grafana grafana/grafana-enterprise:<version number>
+```
 
-# starts grafana with your user id and using the data folder
-docker run -d --user $ID --volume "$PWD/data:/var/lib/grafana" -p 3000:3000 grafana/grafana-enterprise:8.2.1
+Example:
+
+This command will run Grafana Enterprise container which has version 9.4.7. If you want to run a different version e.g. 9.4.3 then just put it in the version-number section
+
+```bash
+docker run -d -p 3000:3000 --name grafana grafana/grafana-enterprise:9.4.7
 ```
 
-## Default paths
+## Run the Grafana main branch
+
+For every successful build of the main branch, we update the `grafana/grafana-oss:main` and `grafana/grafana-oss:main-ubuntu` tags. Additionally, two new tags are created, `grafana/grafana-oss-dev:<version>-<build ID>pre` and `grafana/grafana-oss-dev:<version>-<build ID>pre-ubuntu`, where version is the next version of Grafana and build ID is the ID of the corresponding CI build. You can use these Tags to get access to the latest main builds of Grafana.
```

Comment:
```suggestion
After every successful build of the main branch, two tags, `grafana/grafana-oss:main` and `grafana/grafana-oss:main-ubuntu`, are updated. Additionally, two new tags are created: `grafana/grafana-oss-dev:-pre` and `grafana/grafana-oss-dev:-pre-ubuntu`, where version is the next version of Grafana and build ID is the ID of the corresponding CI build. You can use these tags to get access to the latest main builds of Grafana.
```

Reply:
I don't quite understand what "where version is the next version of Grafana and build ID is the ID of the corresponding CI build." means - I don't see version or build ID in the tags.

Reply:
These need to be rechecked by a developer as it is taken from the original (aka current documentation) and I did not do any modification. To what I understand that it is about the docker images available on
https://hub.docker.com/layers/grafana/grafana-oss/main-ubuntu/images/sha256-d42e791118547736a3bea01adb978055bad6d0cfee63914e37737ff1e12f9d8d

and 

https://hub.docker.com/layers/grafana/grafana-oss/main/images/sha256-43b0cf88ad64b3947d0f41654070ae0189b7a44674abbdcd22052053b28e1c19?context=explore

Dan might also help here.

Reply:
Acknowledged, thanks.

Reply:
`<version>` and `<build ID>` are placeholders, if you look at the docker repo you can see the tags like [10.1.0-55763pre](https://hub.docker.com/layers/grafana/grafana-enterprise-dev/10.1.0-55763pre/images/sha256-b13066ff05063e3b7e4b9128ce93398858711b068abde717de1df2f37d45eb0b?context=explore) (where version is 10.1.0 and build id is 55763)

The main and main-ubuntu tags in the main grafana-oss repo are used to run the latest build from the main branch, the specific tags in the grafana-oss-dev repo are used if you want to run unreleased versions but control which specific build you want.  Neither of these are really useful to a typical end user though as in both cases you're running on the bleeding edge.